build_test.go

```diff
@@ -28,8 +29,10 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		cnbPath    string
 		timestamp  string
 
+		entryResolver     *fakes.EntryResolver
```

Comment:
Looks like this just needs `go generate` run

Reply:
Not sure how the tests passed locally... maybe we forgot to stage the fake directory SMH. Thanks!